internal/schema/schema.go

```diff
@@ -822,6 +841,19 @@ func (s *schemaFetcher) buildTable(
 			}
 		}
 
+		var identity *ColumnIdentity
+		if len(column.IdentityType) > 0 {
+			identity = &ColumnIdentity{
+				Type:       ColumnIdentityType(column.IdentityType),
```

Comment:
Should there be an error check here in case an unexpected type is returned?

Reply:
No! This is very intentional. This enables us to diff schemas without erroring out if the column identity doesn 't change. You could imagine Postgres adds a new column identity. If we errored here, our tooling would totally break even if the column doesn't change. Instead, we should aim to error out within the SQL generator, i.e., only if what we don't know about **changes**